.env.sample

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_API_URL=http://localhost:4000/api/v1
+#Optional
+#NEXT_PUBLIC_MOCK_API=TRUE
```

Comment:
Do we really need this kind of comments left out? Just remove it, if we want this constant we can add it. And if we want it configurable then we should make a way to configure it I think...

Reply:
Sure, I'll remove the comments but keep the variables there. This file is only used to easily generate a `.env.local` by copying it